lib/sls-api-file-relation.js

```diff
@@ -108,10 +119,14 @@ module.exports = class SlsApiFileRelation extends API {
 
 		const [entityId] = this.pathParameters;
 
-		const { fileName: name, fileSource: path, ...additionalData } = this.data;
+		const { fileName: name, fileSource: path, expiration, ...additionalData } = this.data;
 
 		const { ContentLength, ContentType } = await this.getFileInfo(path);
 
+		const expirationOption = EXPIRATION_FILE_OPTIONS[expiration] || EXPIRATION_FILE_OPTIONS[DEFAULT_EXPIRATION_FILE];
+
+		const expirationDate = this.getExpirationDate(expirationOption);
```

Comment:
Si no usas expirationOption en otro lado que como parámetro para `this.getExpirationDate` tonce pasale el `expiration` como parámetro y que el metodo sea el que saque el `expirationOption`, asi la variable muere ahi y ademas no mezclamos cosas que podrian estar "aisladas" en metodos